tests/Unit/Helpers/DataStructures/Tensor/Expressions/EvaluateRank1TestHelpers.hpp

```diff
@@ -13,6 +13,9 @@
 #include "Utilities/GenerateInstantiations.hpp"
 #include "Utilities/TMPL.hpp"
 
+namespace TestHelpers {
```

Comment:
clang-tidy will want you to do `TestHelpers::TensorExpressions`. This type of namespace scoping is a C++17 feature :)